tutorials/darcy_thermo_mech/doc/content/workshop/numerical/fvm_overview.md

```diff
@@ -0,0 +1,245 @@
+# New Feature: Finite Volume Method (FVM)
+
+!---
+
+## Advantages of Finite Volume Method
+
++(1) Flexible:+ Works with arbitrary cell types (even if they are mixed in the same problem)
+
++(2) Inherently conservative:+ Uses the integral form of the PDEs. This makes it perfect for
+      applications where conservation laws need to be respected (fluid flows).
+
++(3) "Accurate":+ Can be second-order accurate in space.
+
++(4) Rich literature:+ Widely used in Computational Fluid Dynamics (CFD) and has been researched for decades.
+
+!---
+
+## Approximating the Solution
+
+!row!
+!col! width=50%
+
+!style fontsize=80%
+First, domain $\Omega$ is split into $N_c$ cells ($\Omega_{i},~i=1,...,N_c$).
+
+!media darcy_thermo_mech/fvm_cell.png style=width:100%;background:white;
+
+!col-end!
+
+!col! width=50%
+
+In MOOSE, the cell-centered finite volume approximation ([!citet](moukalled2016finite) and [!citet](jasak1996error)) is used:
```

Comment:
Should we mention that we use cell-centered approaches because it seamlessly supports both structured and unstructured meshes?

Reply:
Great idea!